src/contentsharecontroller/DefaultContentShareController.ts

```diff
@@ -147,7 +147,7 @@ export default class DefaultContentShareController
           return;
         }
         const stream = this.mediaStreamBroker.mediaStream;
-        if (stream.getVideoTracks().length > 0) {
+        if (stream && stream.getVideoTracks().length > 0) {
```

Comment:
This can be more simply worded as

```
if (stream?.getVideoTracks().length) {
```

Reply:
Totally, I have made the modification. Thanks for your feedback @richnew10 !